config/TLS-example/docker-compose.yml

```
@@ -1,7 +1,7 @@
 services:
   auth:
     container_name: auth
-    image: ghcr.io/neicnordic/sensitive-data-archive:v0.2.23-auth
+    image: ghcr.io/neicnordic/sensitive-data-archive:v3.0.14-auth
```

```suggestion
    image: ghcr.io/neicnordic/sensitive-data-archive:v3.0.14
```

Auth doesn't have a separate image anymore.

---

Updated to use `ghcr.io/neicnordic/sensitive-data-archive:v3.0.14` with `command: sda-auth`, is there any env vars that we need to populate as well?